### PR TITLE
Datadog self hosted

### PR DESCRIPTION
**Pull request by taroface**
Extends #12220.

Various tweaks to the Datadog integration for self-hosted clusters, incorporating the PR opened by @gwaldo.

### CODE REVIEW

**Comment by gwaldo** [comment]
👍

File: v21.2/datadog.md [hunk 1]
```diff
@@ -90,7 +90,9 @@ Log collection is disabled by default in the Datadog Agent. Follow the steps in
 
 ## Step 3. Verify Datadog connection to CockroachDB
 
-Run the Datadog [`status` subcommand](https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6v7#agent-information) for your platform to verify that the CockroachDB integration is properly connected.
+To apply your configuration changes, ensure that you have restarted the Datadog Agent.
+
+Run the Datadog Agent [`status` subcommand](https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6v7#agent-information) for your platform to verify that the CockroachDB integration is properly connected.
```
Comment (kathancox) [inline v21.2/datadog.md:95]: Just a note: here "integration" is not capitalized, as part of "CockroachDB Integration" in the Step 1 title it is capitalized — is this a specific tool?

**Review by taroface — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13312)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kathancox and @MichaelTrestman)

___
*[v21.2/datadog.md, line 95 at r2](https://reviewable.io/reviews/cockroachdb/docs/13312#-MympmjvbfBj8V3bqXcA-r2-95:-MyxqzYC5yhye4awMCQ6:b95qwsx) ([raw file](https://github.com/cockroachdb/docs/blob/35d709c252736ceee8a0ca1370eadb42ee5de6f9/v21.2/datadog.md#L95)):*
<details><summary><i>Previously, kathancox (Kathryn Hancox) wrote…</i></summary><blockquote>

Just a note: here "integration" is not capitalized, as part of "CockroachDB Integration" in the Step 1 title it is capitalized — is this a specific tool? 
</blockquote></details>

Thanks for catching this - I believe it can be lowercase, so changing throughout.
___
*[v21.2/datadog.md, line 131 at r2](https://reviewable.io/reviews/cockroachdb/docs/13312#-MympmjvbfBj8V3bqXcA-r2-131:-MyxrIXFE7cO4pCLSWJJ:b-cksny) ([raw file](https://github.com/cockroachdb/docs/blob/35d709c252736ceee8a0ca1370eadb42ee5de6f9/v21.2/datadog.md#L131)):*
<details><summary><i>Previously, kathancox (Kathryn Hancox) wrote…</i></summary><blockquote>

Note: "Integration" capped here 

Also, looks like `CockroachDB Overview` is a UI title in the screenshot, maybe bold that per style guide?
</blockquote></details>

Hmm I believe it's the name of the pre-built dashboard, not an official UI element - so code formatting seems best here.

**Review by MichaelTrestman — APPROVED** [review]
:lgtm:

> Reviewed 3 of 3 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13312#-:-Mz2qCg65ehdg2Ug2ezR:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @kathancox)

**Pull request merged**